util/omrutil/unix/zos/64/j9memclrz10_64.s

```diff
@@ -18,7 +18,7 @@
 * [1] https://www.gnu.org/software/classpath/license.html
 * [2] http://openjdk.java.net/legal/assembly-exception.html
 *
-* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
```

Comment:
This will need to be broken up into two lines as on z/OS we hit the 71 character limit for the assembler. Similarly in all other z/OS assembler files and any .m4 files which will generate assembler files.

Reply:
Also do we need copyright date updates for license changes?

Reply:
blah, good points. Thanks @fjeremic !